types/mux-embed.d.ts

```diff
@@ -548,6 +548,16 @@ declare module 'mux-embed' {
 
   export const events: MuxEvents;
 
+  type MuxType = (task: any) => void;
+  type muxProps = {
```

Comment:
nitpick(non-blocking): We should be able to just add every named `export` defined in this ambient module to the `muxProps` type to avoid any future surprises.

Reply:
Updated.